### PR TITLE
Prepare hw_2 for wdio

### PR DESCRIPTION
**Pull request by danikkrapivnitskiy**

### CODE REVIEW

File: Automation/wdio/practice/src/tests/hw_2/background.test.ts [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+            interval: 1000,
+            reverse: true
+        });
+
+       const user = await $(userId)
+       await user.waitForDisplayed({
+            timeoutMsg: 'User is logged!',
+            timeout: 5000,
+            interval: 1000,
+            reverse: false
```
Comment (anatoly-karpovich) [inline Automation/wdio/practice/src/tests/hw_2/background.test.ts]: You can specify only needed values, others can stay default)

File: Automation/wdio/practice/src/tests/hw_2/background.test.ts [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+       await (await $(spinnerLocator)).waitForDisplayed({
+            timeoutMsg: 'Spinner is still present!',
+            timeout: 5000,
+            interval: 1000,
+            reverse: true
+        });
+
+       const user = await $(userId)
+       await user.waitForDisplayed({
```
Comment (anatoly-karpovich) [inline Automation/wdio/practice/src/tests/hw_2/background.test.ts]: No need to await userName, it must be displayed after spinner gone

File: Automation/wdio/practice/src/tests/hw_2/background.test.ts [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+        for (let element of lefMenuBarElements) {
+            await element.click()
+            await browser.waitUntil(async () => {
+                const color = await element.getCSSProperty('background-color')
+                return color.value !== 'rgba(0,0,0,0)'
+            }, {
+                timeoutMsg: `Element "${element.getAttribute}" wasn't clicked`,
+                timeout: 3000,
+                interval: 1000,
+            })
```
Comment (anatoly-karpovich) [inline Automation/wdio/practice/src/tests/hw_2/background.test.ts]: You can just wait for spinner to hide before cheching the background color

File: Automation/wdio/practice/src/tests/hw_2/heroku.test.ts [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+        
+        await waitForElementWithText(finishSelector, 'Hello World!', 20000)
+    })
+
+    async function waitForElementWithText(selector: string, text: string, timeout: number) {
+        await browser.waitUntil(async () => {
+            const element = await $(selector)
+            const isEnabled = await element.isEnabled();
+            if (isEnabled) {
+                await expect(element).toHaveText(text)
```
Comment (anatoly-karpovich) [inline Automation/wdio/practice/src/tests/hw_2/heroku.test.ts]: no need to have expect here. If such element would not be found - an error will be thrown by waitUntil method

**Review by anatoly-karpovich — APPROVED** [review]
Good!

File: Automation/wdio/practice/src/tests/hw_2/heroku.test.ts (hunk 4 above)
Comment (danikkrapivnitskiy) [reply Automation/wdio/practice/src/tests/hw_2/heroku.test.ts]: but I need to verify text in element

Comment (anatoly-karpovich) [reply Automation/wdio/practice/src/tests/hw_2/heroku.test.ts]: const text = await element.getText()
return text === expectedText

**Review by anatoly-karpovich — APPROVED** [review]

**Pull request merged**